src/bot/saveStats.ts

```diff
@@ -75,9 +84,20 @@ export function statsString(
 ): string {
   const header = `📝 ${stats.name}${stats.table ? ` (${stats.table})` : ""}`;
   const stepsString = steps.map((s) => `\t${s}`).join("\n");
+  const skipped = getSkippedCount(stats);
```

Comment:
Move to it's own function `skippedString(stats: SaveStats): string` and add some tests